server/tests/api/server/stats.ts

```diff
@@ -135,6 +142,68 @@ describe('Test stats (excluding redundancy)', function () {
     }
   })
 
+  it('Should have the correct active channel stats', async function () {
+    const server = servers[0]
+    const videoChannelProperties = { name: 'second_channel', displayName: 'My second channel' }
+
+    {
+      const res = await getStats(server.url)
+      const data: ServerStats = res.body
+      expect(data.totalLocalDailyActiveVideoChannels).to.equal(1)
+      expect(data.totalLocalWeeklyActiveVideoChannels).to.equal(1)
+      expect(data.totalLocalMonthlyActiveVideoChannels).to.equal(1)
+    }
+
+    {
+      await addVideoChannel(server.url, server.accessToken, videoChannelProperties)
+
+      const res = await getStats(server.url)
+      const data: ServerStats = res.body
+      expect(data.totalLocalDailyActiveVideoChannels).to.equal(1)
+      expect(data.totalLocalWeeklyActiveVideoChannels).to.equal(1)
+      expect(data.totalLocalMonthlyActiveVideoChannels).to.equal(1)
+    }
+
+    {
+      const resChannel = await getMyUserInformation(server.url, server.accessToken)
+      const videoChannel = server.videoChannel = resChannel.body.videoChannels.find(v => v.name === videoChannelProperties.name)
```

Comment:
Please don't override `server.videoChannel`. Just use a global variable instead. Also, you can use the answer of `addVideoChannel` to get the channel id: https://github.com/Chocobozzz/PeerTube/blob/develop/server/tests/api/search/search-activitypub-videos.ts#L104

This should resolve the test api-2 failure